play-ahc-ws-standalone/src/main/java/play/libs/ws/ahc/StandaloneAhcWSRequest.java

```diff
@@ -331,6 +339,11 @@ public String getMethod() {
         return Optional.ofNullable(this.followRedirects);
     }
 
+    @Override
+    public Optional<Boolean> getDisableUrlEncoding() {
```

Comment:
`Boolean` with default value

Reply:
I agreed at first sight, but 
https://github.com/playframework/play-ws/pull/552/files#diff-86c3e80542b38a0d4fd25d703e094cc820766fff4ed6082e233a0857a5a3c83eR426-R431 makes it clear that we need a `Troolean` (three states). 

If explicit set by user (true or false), we should use it, otherwise we fallback to `isDisableUrlEncodingForBoundRequests`.